app/models/rental.py

```diff
@@ -0,0 +1,21 @@
+from flask import current_app
+from app import db
+# from datetime import datetime, timedelta 
+
+
+class Rental(db.Model):
+    rental_id = db.Column(db.Integer, primary_key=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.customer_id'))
+    video_id = db.Column(db.Integer, db.ForeignKey('video.video_id'))
+    due_date = db.Column(db.DateTime, nullable=True, default=None)
+    #alternate default for due_date: default=(datetime.now()+timedelta(days=7))
+
```

Comment:
You may want to add a check in date to know if the user returned this rental:

```suggestion
    check_in_date = db.Column(db.DateTime, nullable=True)
```